### PR TITLE
#001: Setup common test libs

### PR DESCRIPTION
**Pull request by samflores**
Add and configure the following gems:
- Rspec
- Rubocop
- SimpleCov

### CODE REVIEW

**Comment by lucascarrias** [comment]
LGTM

**Comment by Isaius** [comment]
Closes #1

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  - rubocop-rspec
+  - rubocop-rails
+
+AllCops:
+  NewCops: enable
+
+RSpec/FactoryBot/CreateList:
+  Include:
+    - 'spec/factories.rb'
+    - 'spec/factories/**/*.rb'
```
Comment (samflores) [inline .rubocop.yml:12]: This shouldn't be necessary as it's the default option for this Cop, but for some reason Rubocop was trying to run the cop on the wrong files.

**Review by Isaius — APPROVED** [review]
All good here

**Review by joaoguilherme-devsec — APPROVED** [review]
seems ok to me

**Review by lucascarrias — APPROVED** [review]

**Pull request merged**